packages/graphql-codegen-cli/src/utils/listr-renderer.ts

```diff
@@ -29,7 +29,7 @@ export class Renderer {
   ) {
     this.updateRenderer.end(err);
 
-    if (typeof err === undefined) {
+    if (typeof err === 'undefined') {
```

Comment:
Not sure if this is correct, `err` has an Error type, so cant be `undefined`.
Very suspicious